SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -775,72 +774,68 @@ void CIncNSSolver::SetTauWall_WF(CGeometry *geometry, CSolver **solver_container
       unsigned long counter = 0;
       su2double diff = 1.0;
       su2double U_Tau = max(1.0e-6,sqrt(WallShearStress/Density_Wall));
-      su2double Y_Plus = 5.0; // clipping value
+      su2double Y_Plus = 0.99*config->GetwallModelMinYPlus(); // clipping value
```

Comment:
It should actually be something along the lines of 'use minimum y+ as defined in the config file in case the routine below for computing y+ does not converge.'